confignode/src/main/java/org/apache/iotdb/confignode/persistence/NodeInfo.java

```diff
@@ -268,6 +268,21 @@ public int getRegisteredConfigNodeCount() {
     }
     return result;
   }
+
+  /** Return the number of registered ConfigNodes */
+  public int getRegisteredNodeCount() {
+    int result;
```

Comment:
Change the comment, please. actually, this method calculated the total number of nodes for confignodes and datanodes.

Reply:
Okay, I get it ~